Client/WebFilters/ShieldStats/Adblock/AdBlockStats.swift

```diff
@@ -198,6 +198,15 @@ extension AdBlockStats {
       
       cssRules += "\(key){" + subRules + " !important}\n"
     }
+
+    var injectedScript = model.injectedScript
+
+    if !injectedScript.isEmpty, Preferences.Shields.autoRedirectAMPPages.value {
```

Comment:
Why is `deAmpEnabled = true` only injected if `CosmeticFilters` are injected? Are they not independent of each other?